app/controllers/govuk_publishing_components/component_guide_controller.rb

```diff
@@ -110,5 +111,9 @@ def component_breadcrumb(component_doc, component_example = nil)
         h[:url] = component_doc_path(component_doc.id) if component_example
       end
     end
+
+    def components_used_by_component_guide
+      %w(breadcrumbs details layout-footer layout-header lead-paragraph search textarea title)
```

Comment:
Can we automate these, in a similar way we do to determine which components are used in a certain app?

Reply:
Done, thanks for the suggestion.